### ISSUE TITLE
Crash when driving off an asteroid in SE

### ISSUE THREAD

**Issue by nullspeaker** [azaghal/Factorio-PavementDriveAssistContinued#2]
![image](https://user-images.githubusercontent.com/2228057/208281395-455eadb3-3020-4c51-be1c-6c448f85713e.png)

**Comment by nullspeaker** [comment]
Assist was on, have not tried with assist off.

**Comment by azaghal** [comment]
Hm... That's an interesting crash, to say the least. I'm a bit slower to respond at the moment with bugfixes, but I'll try to have a look at it soon-ish. I am almost wondering if it's possible that no tile exists at that spot and that's the reason for the crash (basically trying to access name on an invalid object) - but it does feel a bit weird - I'd expect to get a null object returned instead.

Thanks for reporting with clear reproduction steps, though, it is much appreciated :)

**Comment by nullspeaker** [comment]
Yeah; I took a peek at it briefly and think I could fix it with a `nil` check but haven't taken the time for it -- holidays have had me pretty tied up lately!

**Comment by azaghal** [comment]
Ok, so finally got around to trying to reproduce this one, and... With current release of Space Exploration I was not able to even place a vehicle on one of the space surfaces (I tried Nauvis orbit), so...

How were you able to place a vehicle in space at all? :) Or maybe it was some older version of Space Exploration?

Basically, I'm trying to reproduce the issue locally so I could fix it.
